src/CoreShop/Bundle/FrontendBundle/Resources/install/pimcore/staticroutes.yml

```diff
@@ -22,6 +22,20 @@ routes:
     variables: _locale,product
     priority: 3
 
+  coreshop_category_list:
```

Comment:
`_list` suffix here?

Reply:
it always was list, so I thought to keep it that way